src/Polytopes/PolyhedralFan/properties.jl

```diff
@@ -361,3 +361,149 @@ true
 ```
 """
 iscomplete(PF::PolyhedralFan) = pm_fan(PF).COMPLETE
+
+
+###############################################################################
+## Primitive collections
+###############################################################################
+
+@doc Markdown.doc"""
+    primitive_collections(PF::PolyhedralFan)
+
+Computes the primitive collections of a polyhedral fan.
+
+# Examples
+```jldoctest
+julia> primitive_collections(normal_fan(Oscar.simplex(3)))
+1-element Vector{Vector{Int64}}:
+ [1, 2, 3, 4]
+```
+"""
+function primitive_collections(PF::PolyhedralFan)
+    # collect data
+    cones = [findall(x->x!=0, l) for l in eachrow(pm_fan(PF).MAXIMAL_CONES)]
+    all_points = [i for i in 1 : pm_fan(PF).N_RAYS]
+    d_max = maximum([length(i) for i in cones]) + 1
+    # identify and return the primitive collections
+    collections = Vector{Int}[];
+    for d in 1 : d_max
+        checked  = Vector{Int}[];
+        for cone in cones
+            if d <= length(cone)
```

Comment:
This code nests 8-9 levels, that's deep, perhaps too deep. Makes it harder to read, but also to follow. Personally, I'd do the following (and then of course get rid of the matching `end` and change the indentation)
```suggestion
            d <= length(cone) || continue
```

Reply:
Thank you for the suggestion @fingolfin . I'll take care of `end` and the indentation.

Reply:
OK. Seems this has not yet happened, which causes the CI failures